_posts/2016-06-09-bayesian-quadrature.markdown

```diff
@@ -2,4 +2,56 @@
 layout: post
 title:  "Bayesian Quadrature"
 categories: jekyll update
+img: bayesian_quadrature.png
```

Comment:
The figure is nice but I am not sure it encapsulates the idea of BQ, specially the subfigure on the right. I leave to you is you like it though.

Reply:
Something like the figure on the left but with a cleared pattern of points capturing the density would be nice. Something on the lines of the figure used in paper 1 here:

https://warwick.ac.uk/fac/sci/statistics/staff/research_students/briol/probabilistic_integration/

Reply:
sounds good. I'll do another PR for this which just replaces the figure.

I'll need a running version of BQ for it and we don't have one right now.